client.go

```diff
@@ -575,6 +579,17 @@ func StartAtEarliestReceived() SubscriptionOption {
 	}
 }
 
+// ReadReplica sets read replica option. If true, the client will request
+// subscription from an ISR replica instead of subscribing to partition's leader
+// there may be cases where replicas do not catch up with partition's leader
+// so use this option with consideration
+func ReadReplica(readReplica bool) SubscriptionOption {
```

Comment:
I think we should remove the bool passed in and make it a no-argument function that sets the flag to `true`. I don't see a need to toggle this since the default behavior is disabled.

Reply:
indeed. Change has been made